packages/node/src/core/evm/fulfillments/api-calls.test.ts

```diff
@@ -23,7 +25,7 @@ import { RequestErrorCode, RequestStatus } from 'src/types';
 describe('submitApiCall', () => {
   const masterHDNode = wallet.getMasterHDNode();
   const gasPrice = ethers.BigNumber.from('1000');
-  const txOpts = { gasLimit: 500_000, gasPrice, nonce: 5 };
+  const txOpts = { gasLimit: GAS_LIMIT, gasPrice, nonce: 5 };
```

Comment:
I was thinking the same thing but since the #148 stated that constants should be provided from same source file, I did as requested. Will revert this.